code/game/vehicles/spacepods/equipment.dm

```diff
@@ -98,6 +103,34 @@
 	shot_cost = 600
 	fire_sound = 'sound/weapons/Laser.ogg'
 
+// MINING LASERS
+/obj/item/device/spacepod_equipment/weaponry/mining_laser_basic
+	name = "weak mining laser system"
+	desc = "A weak mining laser system for space pods, fires bursts of energy that cut through rock"
+	icon_state = "pod_taser"
+	projectile_type = "/obj/item/projectile/kinetic"
+	shot_cost = 200
+	fire_delay = 10
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
+
+/obj/item/device/spacepod_equipment/weaponry/mining_laser
+	name = "mining laser system"
+	desc = "A mining laser system for space pods, fires bursts of energy that cut through rock"
+	icon_state = "pod_m_laser"
+	projectile_type = "/obj/item/projectile/kinetic/hyper"
+	shot_cost = 600
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
+
+/obj/item/device/spacepod_equipment/weaponry/mining_laser_burst
+	name = "burst mining laser system"
+	desc = "A mining laser system for space pods, this one fires 3 at a time"
+	icon_state = "pod_w_laser"
+	projectile_type = "/obj/item/projectile/kinetic/hyper"
+	shot_cost = 1200
+	shots_per = 3
+	fire_delay = 30
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
```

Comment:
This thing is way too good.

It's effectively a 144 damage burst weapon. You're going to see space pod pilots asking for this because of how powerful it is.